src/Tests/SignatureTests.cs

```diff
@@ -210,7 +211,8 @@ bool ValidateReport(SignedReport signedReport)
             }
 #endif
 
-            var reserializedReportBytes = JsonSerializer.SerializeToUtf8Bytes(signedReport.ReportData, SerializationOptions.NotIndentedWithNoEscaping);
+            var reserializedReport = JsonSerializer.Serialize(signedReport.ReportData, SerializationOptions.NotIndentedWithNoEscaping);
+            var reserializedReportBytes = Encoding.UTF8.GetBytes(reserializedReport);
```

Comment:
It's just nice to be able to set a breakpoint on line 215 and find out just how did that serialize to JSON anyway?